test/e2e/framework/util.go

```diff
@@ -2251,40 +2267,46 @@ func KubectlCmd(args ...string) *exec.Cmd {
 	return cmd
 }
 
-// kubectlBuilder is used to build, customize and execute a kubectl Command.
+// KubectlBuilder is used to build, customize and execute a kubectl Command.
 // Add more functions to customize the builder as needed.
-type kubectlBuilder struct {
+type KubectlBuilder struct {
 	cmd     *exec.Cmd
 	timeout <-chan time.Time
 }
 
-func NewKubectlCommand(args ...string) *kubectlBuilder {
-	b := new(kubectlBuilder)
+// GetNewKubectlCommand returns a KubectlBuilder for running kubectl.
+func GetNewKubectlCommand(args ...string) *KubectlBuilder {
```

Comment:
I liked `NewKubectlCommand` actually, was golint complaining about this?

Reply:
Thank you for comment.
I've fixed this.